### PR TITLE
ECDF plot makeover (#1747)

### PR DESCRIPTION
**Pull request by islandblue**

### CODE REVIEW

**Comment by nikohansen** [comment]
The circleci test fails with a problem that I already fixed with commit 3383cd1540034b030367cef116cd4ce78f970fc4 in the development branch. I don't quite understand why this is happening. Any idea?

**Comment by islandblue** [comment]
`test-postprocessing` failed on Jenkins

> IOError: [Errno 28] No space left on device

What to do? Request more space? Try to clean up?

File: code-postprocessing/cocopp/compall/pprldmany.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
 from pdb import set_trace
 import numpy as np
 import matplotlib.pyplot as plt
+from matplotlib.ticker import (FixedLocator, FuncFormatter, NullFormatter)
```
Comment (nikohansen) [inline code-postprocessing/cocopp/compall/pprldmany.py]: minor comment: using braces here doesn't look like python coding style.

Comment (islandblue) [reply code-postprocessing/cocopp/compall/pprldmany.py]: Busted! I copied the code from [here](https://matplotlib.org/gallery/ticks_and_spines/major_minor_demo.html#major-minor-demo) and adapted it. I'll make a note for future work. Thanks for the comment.

**Comment by brockho** [comment]
I now removed the texlive installation on our linux Jenkins slave. Since we do not run anymore the LaTeX tests anyway this was a (relatively) save move :-) All tests seem to run through again.

We have currently 16% of the main disk free (unfortunately only 2.9GB) which should give us some slack for the next weeks/months to come.

**Pull request merged**